README.md

```diff
@@ -15,7 +15,8 @@ cargo install --locked --git https://github.com/r0gue-io/pop-cli
 > :information_source: For detailed instructions on how to install Pop CLI, please refer to our
 > documentation: https://learn.onpop.io/v/contracts/welcome/installing-pop-cli
```

Comment:
Why is this listed under contracts? Surely it would be better to have this defined once under https://learn.onpop.io/v/cli rather than having it defined under each section (contracts/appchains), creating additional maintenance burden if anything needs to change.

Fine to have a pop-cli page under contracts/app-chains, but they should link to CLI section for specifics so important things are only defined in one place.

Reply:
Done.